motor/motor_gridfs.py

```diff
@@ -213,14 +210,12 @@ def open(self):
            ``open`` explicitly is rarely needed.
         """
         return self._framework.chain_return_value(self._ensure_file(),
-                                                  self.get_io_loop(),
-                                                  self)
+                                                  self.get_io_loop())
```

Comment:
Missing the third argument here: `self`

Reply:
Thanks, I was just chasing down the failures myself.